SwiftCSV/NamedCSVView.swift

```diff
@@ -35,19 +35,10 @@ public struct Named: CSVView {
     }
 
     public func serialize(header: [String], delimiter: CSVDelimiter) -> String {
-        let separator = String(delimiter.rawValue)
+        let mappedRows = rows.map { row in
```

Comment:
What would be a better name to describe the result this produces? "Mapped rows" is a code smell describing `rows.map` is being called.

The odd thing for a reader, and it was a similar problem in the past, is that for each row, something is done on for each header element. It read as if this is inverted. Not much we can do since lines are produced top-down.

If you have no idea, either, maybe add a comment to guide the reader?

```swift
/// The order of cells is determined by `header`: we serialize the unsorted `rows` dictionary using the sequential `header` to put all cells in order.
```

Feel free to edit!

And sorry this 2016 port is not in an ideal state 😅

Reply:
I've come up with `rowsOrderingCellsByHeader`, which deliberately implies that the code is an elision of:

```Swift
let rowsOrderingCellsByHeader = rows.map { row in
    row.orderingCells(by: header)
}

extension Named.Row {
    func orderingCells(by header: [String]) -> [String] {
        return header.map { cellID in self[cellID]! }
    }
}
```